src/components/frame/Extensions/layouts/Header.tsx

```diff
@@ -35,7 +36,10 @@ const Header: FC<HeaderProps> = ({ navBar = null, txButtons, userHub }) => {
   const userHubComponent = userHub || <HeaderAvatar />;
 
   const userMenuComponent = isActionSidebarOpen ? (
-    <ActionSidebar transactionId={transactionId || undefined}>
+    <ActionSidebar
+      transactionId={transactionId || undefined}
```

Comment:
Should we pass undefined here?

Reply:
![image](https://github.com/JoinColony/colonyCDapp/assets/38806520/61606f9e-7678-446e-b517-56c48de7093b)
`transationId` is nullable so yes